code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -107,10 +108,10 @@ Difficulty: Hard
 	for(var/mob/living/simple_animal/hostile/megafauna/bubblegum/B in GLOB.mob_living_list)
 		if(B != src)
 			return INITIALIZE_HINT_QDEL //There can be only one
-	var/obj/effect/proc_holder/spell/bloodcrawl/bloodspell = new
+	/*var/obj/effect/proc_holder/spell/bloodcrawl/bloodspell = new
```

Comment:
Mark your changes ![like this](https://github.com/yogstation13/tg-rebase/blob/master/yogstation/README.md)